src/App.tsx

```diff
@@ -58,8 +59,19 @@ const App = (props: AppProps): JSX.Element => {
 
   useEffect(() => {
     initialState();
+    changeLanguage();
   }, []);
 
+  const changeLanguage = () => {
+    const websiteLanguage = getCookie('LOCALE');
+
+    if (websiteLanguage && websiteLanguage === 'zh') {
+      i18next.changeLanguage('cn');
+    } else if (websiteLanguage && websiteLanguage != 'zh') {
+      i18next.changeLanguage(websiteLanguage);
+    }
+  };
```

Comment:
Put this function in the service, so that everything related to translations is kept there

Reply:
Done